features/toolbar/components/styles/Styles.js

```diff
@@ -19,15 +19,16 @@ export const styles = createStyleSheet({
      */
     toolbarButton: {
         alignSelf: 'center',
-        backgroundColor: ColorPalette.jitsiBlue,
+        backgroundColor: 'white',
```

Comment:
I was thinking about this, but it's just a white, so wasn't sure why we would do that. I may change it to toolbarButtonColor, then it will make sense, but I don't want us to be very detailed about where we use a particular color in the ColorPalette.